server/resolvers/browsersResolver.js

```diff
@@ -1,7 +1,7 @@
 const { getBrowsers } = require('../models/services/BrowserService');
 
 const browsersResolver = () => {
-    return getBrowsers(undefined, undefined, undefined, undefined, {
+    return getBrowsers(undefined, undefined, undefined, undefined, undefined, {
         order: [['name', 'asc']]
     });
```

Comment:
The previous version seems correct to me! Are you sure the change is needed?

Reply:
Good catch! After looking at the `getBrowsers` API, it is not needed.